packages/patternfly-4/react-integration/cypress/integration/clipboardcopy.spec.ts

```diff
@@ -0,0 +1,16 @@
+describe('Chip Group Demo Test', () => {
+  it('Navigate to demo section', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('#clipboard-copy-demo-nav-item-link').click(); 
+    cy.url().should('eq', 'http://localhost:3000/clipboard-copy-demo-nav-link');
+  });
+
+  it('Verify form input', () => {
+    cy.get('input').should('have.class', 'pf-c-form-control');
+  });
+  
+  it('Verify content expands', () => {
+    cy.get('.pf-c-clipboard-copy__group-toggle').click(); 
+    cy.get('.pf-c-clipboard-copy').should('have.class', 'pf-m-expanded');
+  });
```

Comment:
I think we should add more verification steps here.  We dod not verify that the copy actually works and I think we need to.

Reply:
Do you know of a good way to test the copy paste feature? I tried this: 
```js
it('Verify copy to clipboard works', () => {
    const form = cy.get('.pf-c-form-control');
    form.clear().type('Hi'); 
    cy.get('.pf-c-clipboard-copy__group-copy').click(); 
    cy.get('.pf-c-form-control').clear().type('{cmd}v');
    form.should('have.value', 'Hi'); 
  });
```
But it always ends up typing "v" instead of doing a paste. I've been looking into it, but I'm not sure what the solution would be: https://github.com/cypress-io/cypress/issues/3316